src/ghost_story_factory/pregenerator/state_hasher.py

```
@@ -0,0 +1,122 @@
+"""
+状态哈希计算器
+
+用于对话树生成过程中的状态去重，识别重复的游戏状态并合并节点，
```

Ruff (0.14.8)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)